packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
-import { findDOMNode } from 'react-dom';
 import { BaseDecorator } from './BaseDecorator';
-import { getWindow, hoistStatics } from '../../Utilities';
+import { EventGroup, getWindow, css } from '@uifabric/utilities';
+import * as WithResponsiveModeStyles from './withResponsiveMode.scss';
```

Comment:
Since this is only one very simple style, can you just use inline styles instead of SCSS?

Reply:
Shouldn't we standardize one way or the other? If I did an inline style, I'd still wind up putting it into a module-level `const`, which is equivalent to using CSS modules, I think.